index.bs

```diff
@@ -213,30 +218,41 @@ The <dfn>AV1CodecConfigurationBox</dfn> contains decoder configuration informati
 
 ```
 class AV1CodecConfigurationBox extends FullBox('av1C', version = 0, 0){
-  AV1CodecConfigurationRecord av1Config;
+  AV1CodecConfigurationRecordv1 av1Config;
 }
 
-aligned (8) class AV1CodecConfigurationRecord {
-  unsigned int (3) reserved = 0;
+aligned (8) class AV1CodecConfigurationRecordv1 {
+  unsigned int (4) reserved = 0;
+  unsigned int (3) version;
```

Comment:
As said in the text, the syntax should say `version = 1;`

Reply:
Done.